securing_apps/topics/client-registration/client-registration-cli.adoc

```diff
@@ -1,395 +1,360 @@
+The following example shows how to read a JSON file, override any client ID it may contain, set any other attributes, and print the configuration to a standard output after successful creation.
 
-On Linux:
-[source,bash]
+* Linux:
+[options="nowrap"]
 ----
 $ kcreg.sh create -f client-template.json -s clientId=myclient -s baseUrl=/myclient -s 'redirectUris=["/myclient/*"]' -o
 ----
-
-On Windows:
-[source,bash]
+* Windows:
+[options="nowrap"]
 ----
 C:\> kcreg create -f client-template.json -s clientId=myclient -s baseUrl=/myclient -s "redirectUris=[\"/myclient/*\"]" -o
 ----
 
+Run the [command]`kcreg create --help` for more information about the [command]`kcreg create` command.
 
-See `kcreg create --help` for more information about `kcreg create`.
+You can use [command]`kcreg attrs` to list available attributes. Keep in mind that many configuration attributes are not checked for validity or consistency. It is up to you to specify proper values. Remember that you should not have any ID fields in your
+template and should not specify them as arguments to the [command]`kcreg create` command.
 
 
-You can use `kcreg attrs` to list available attributes. Keep in mind that many configuration attributes are not checked for
-validity or consistency. It is up to you to specify proper values. Also note that you should not have any `id` fields in your
-template and should not specify them as arguments to `kcreg create`.
+==== Retrieving a client configuration
 
+You can retrieve an existing client by using the [command]`kcreg get` command.
 
-==== Retrieving client configuration
+For example, on:
 
-You can retrieve an existing client by using `kcreg get`.
-
-For example, on Linux:
-[source,bash]
+* Linux:
+[options="nowrap"]
 ----
 $ kcreg.sh get myclient
 ----
-
-On Windows:
-[source,bash]
+* Windows:
+[options="nowrap"]
 ----
 C:\> kcreg get myclient
 ----
 
+You can also see the client configuration as an adapter configuration file, which you can package with your web application.
 
-You can also get client configuration as adapter configuration file which you can package with your web application.
+For example, on:
 
-For example, on Linux:
-[source,bash]
+* Linux:
+[options="nowrap"]
 ----
 $ kcreg.sh get myclient -e install > keycloak.json
 ----
-
-On Windows:
-[source,bash]
+* Windows:
+[options="nowrap"]
 ----
 C:\> kcreg get myclient -e install > keycloak.json
 ----
 
-See `kcreg get --help` for more information about `kcreg get`.
+Run the [command]`kcreg get --help` command for more information about the [command]`kcreg get` command.
+
+
+==== Modifying a client configuration
 
-==== Modifying client configuration
+There are two methods for updating a client configuration.
 
-There are two modes of updating client configuration.
+One method submits a complete new state to the server after getting the current configuration, saving it to a file, editing it, and posting it back to the server.
```

Comment:
I'm not sure about this sentence. It makes one think that all this happens automatically when it does not. I was hoping the original sentence made it clear that it was a manual process. First you get config and save it into a file, then you edit it, the you post it back. Thus, it is not the 'method' that submits a new state - it's the user that does it. I would prefer: 'One method is to submit ...'.